lib/model/game_pile.dart

```diff
@@ -0,0 +1,10 @@
+enum GamePile {
+  DECK,
+  DISCARD
+}
+
+extension GamePileExt on GamePile {
+  String get description {
+    return this == GamePile.DECK ? "Deck" : "Discard";
+  }
+}
```

Comment:
Hmm, not sure this `enum` is adding a whole load of value. We could just pass a String to the `card_pile` instead, and render it directly?